roles/openshift_health_checker/test/docker_storage_test.py

```diff
@@ -219,6 +241,52 @@ def test_convert_to_bytes(string, expect_bytes):
 ])
 def test_convert_to_bytes_error(string):
     with pytest.raises(ValueError) as err:
-        DockerStorage._convert_to_bytes(string)
+        DockerStorage.convert_to_bytes(string)
     assert "Cannot convert" in str(err.value)
     assert string in str(err.value)
+
+
+ansible_mounts_enough = [{
+    'mount': '/var/lib/docker',
+    'size_available': 50 * 10**9,
+    'size_total': 50 * 10**9,
+}]
+ansible_mounts_not_enough = [{
+    'mount': '/var/lib/docker',
+    'size_available': 0,
+    'size_total': 50 * 10**9,
+}]
+
+
+@pytest.mark.parametrize('ansible_mounts, threshold, expect_fail, expect_msg', [
+    (
+        ansible_mounts_enough,
+        None,
+        False,
+        [],
+    ),
+    (
+        ansible_mounts_not_enough,
+        None,
+        True,
+        ["usage percentage", "higher than threshold"],
+    ),
+    (
+        ansible_mounts_not_enough,
+        "bogus percent",
+        True,
+        ["is not a percentage"],
+    ),
+])
+def test_overlay_usage(ansible_mounts, threshold, expect_fail, expect_msg):
+    check = dummy_check()
+    task_vars = non_atomic_task_vars.copy()
```

Comment:
It's not a big deal as is now. The problem would only show itself as we make changes. If for some reason a test mutates `non_atomic_task_vars`, the tests that run after would start making copies of the mutated value.

If we make it a function that returns the value, we guarantee that we won't introduce that sort of bug.

```python
def non_atomic_task_vars():
    return {"openshift": {"common": {"is_atomic": False}}}

def test_overlay_usage(...):
    # ...
    task_vars = non_atomic_task_vars()
    # ...
```

Making it a fixture or not is orthogonal to that.